modules/nf-core/emu/abundance/meta.yml

```diff
@@ -34,7 +34,7 @@ output:
         Groovy Map containing sample information
         e.g. [ id:'test', single_end:false ]
       pattern: "*{.tsv}"
-  - results/*.tsv:
+  - '*.tsv':
```

Comment:
This meta for the TSV itself is still copy-pasta- from meta

Reply:
Updated the PR.